cns/middlewares/k8sSwiftV2_windows_test.go

```diff
@@ -0,0 +1,41 @@
+package middlewares
+
+import (
+	"testing"
+
+	"github.com/Azure/azure-container-networking/cns"
+	"github.com/Azure/azure-container-networking/cns/middlewares/mock"
+	"gotest.tools/v3/assert"
+)
+
+func TestSetRoutesSuccess(t *testing.T) {
```

Comment:
Do we want this test to be ran in our PR pipeline? Currently the only windows unit tests being run are npm.

CC: @huntergregory @matmerr

Reply:
#1554

Reply:
It's ok with us (NPM team) if you run these CNS UTs in the pipeline job for Windows UTs (which only runs NPM tests right now).